### ISSUE TITLE
Don't autostart when certain apps are running

### ISSUE THREAD

**Issue by firebirdberlin** [firebirdberlin/nightdream#202]
"I have a proposal to improve the application. I have set the 
application to start when the charger is connected and this function is 
just great. However, it would be nice to add a new function in the menu 
that would block the clock from starting when another application is 
running. I am thinking specifically about the navigation application, 
because in the car we use navigation while charging the phone. This way, 
the clock would start at home when the charger is plugged in and stay 
off in the car. "

**Comment by mastermichi** [comment]
Maybe we can solve this problem by starting the app only by active wifi. This could be a new option in the settings.

**Comment by firebirdberlin** [comment]
Possible but then the app needs permissions to access locations. I'd prefer a solution which doesn't need special permissions.

**Comment by firebirdberlin** [comment]
True, this works if you are connected to a wireless network. Anyway, this method will fail in flight mode.
